doc/classes/ItemList.xml

```diff
@@ -333,6 +333,13 @@
 				Sorts items in the list by their text.
 			</description>
 		</method>
+		<method name="update_for_items">
+			<return type="void" />
+			<description>
+				Updates the list's auto height value according to items in the list. If [member auto_height] is [code]true[/code], [method Control.get_minimum_size] will return the current the auto height.
+				Updates scrollbar according to all its items if neccessary.
```

Comment:
```suggestion
				Updates the list's auto height value based on its items. If [member auto_height] is [code]true[/code], [method Control.get_minimum_size] will return the current the auto height.
				Scrollbar is adjusted accordingly if neccessary.
```
The second sentence needs some rewording, because it's not necessarily true. `custom_minimum_size` also affects minimum size.

Reply:
Reworded it a bit to try and account for only returning the auto height value if the auto height is less than the `custom_minimum_size`.